tagstudio/src/qt/ts_qt.py

```diff
@@ -323,7 +323,7 @@ def start(self):
 		# self.entry_panel.setWindowIcon(icon)
 
 		if os.name == 'nt':
-			appid = "cyanvoxel.tagstudio.9"
+			appid = "cyanvoxel.9"
```

Comment:
not sure if this was intentional or a replace all gone wrong.

Reply:
It was a replace gone wrong, good catch. I reverted it.